src/main/kotlin/inc/evil/d17map/controllers/ClassroomController.kt

```diff
@@ -102,13 +98,10 @@ class ClassroomController(private val classroomService: ClassroomService) {
             ApiResponse(responseCode = "200", description = "Successfully updated the classroom."),
             ApiResponse(responseCode = "404", description = "Classroom with the given ID not found."),
             ApiResponse(responseCode = "400", description = "Invalid classroom data."),
-            ApiResponse(
-                responseCode = "401",
-                description = "Unauthorized access. The user is not authenticated and needs to log in."
-            )
+            ApiResponse(responseCode = "401", description = "Unauthorized access. The user is not authenticated and needs to log in.")
         ]
     )
-    @PutMapping("/{buildingName}/{floorName}/admin/{id}")
+    @PutMapping("$BUILDINGS_PATH$FLOORS_PATH$CLASSROOMS_PATH/admin/{id}")
```

Comment:
Wiesz co, ja to bym jednak przeniósł floor do requesta, bo mogłoby się zdarzyć, że jakaś salka o nazwie np. "Sala Chemiczna" zostanie przeniesiona piętro niżej i wtedy taki endpoint wiesz wskazuje ci klasę na piętrze np. 5, a po wykonaniu tego requesta już będzie to endpoint /4 (bo floor jest o 1 niżej).

Najlepiej w tym kontrolerze wszędzie gdzie jest PUT / POST to dajmy floor do requesta i nie będziemy wtedy w endpoincie mieli tego.